cmd/livepeer_cli/wizard.go

```diff
@@ -41,6 +42,23 @@ func (w *wizard) readString() string {
 	}
 }
 
+func (w *wizard) readMultilineString() string {
+	ss := make([]string, 1)
+	fmt.Printf("(press enter, then ctrl+d when done) > ")
+	for {
+		text, err := w.in.ReadString('\n')
+		if err != nil {
+			if err != io.EOF {
+				log.Crit("Failed to read user input", "err", err)
+				continue
+			}
+			break
```

Comment:
If the user presses ctrl+d, we should still read the current line even if there is no newline character at the end. Otherwise, we will always omit the last line of a multi-line message if the last line does not contain a newline.

For example, consider the 3Box auth message:

"Create a new 3Box profile\n\n-\nYour unique profile ID is did:muport:QmPXNzBdKMrHcnEaCwCAGcxGCmyiNqoHYgSn34z5uJFvwh \nTimestamp: 1580338635"

There is no newline at the end of the message. So, when the message is copy + pasted, if the user presses ctrl+d, the parsed message will just be:

"Create a new 3Box profile\n\n-\nYour unique profile ID is did:muport:QmPXNzBdKMrHcnEaCwCAGcxGCmyiNqoHYgSn34z5uJFvwh \n"

And if the user presses enter and *then* ctrl+d, the parsed message will be:

""Create a new 3Box profile\n\n-\nYour unique profile ID is did:muport:QmPXNzBdKMrHcnEaCwCAGcxGCmyiNqoHYgSn34z5uJFvwh \nTimestamp: 1580338635\n"

which is also incorrect because there is an extra newline at the end of the message.

Reply:
This should do the trick without having to join strings together or anything, however in my case I would have to press ctrl+d twice to send an EOF signal without having to create a new line. 

output using `%q` formatter 
```
Enter or paste the message to sign:
(press ctrl+d twice when done) > Create a new 3Box profile

-
Your unique profile ID is did:muport:<DID>
Timestamp: <TIMESTAMP>^D
"Create a new 3Box profile\n\n-\nYour unique profile ID is did:muport:<DID>\nTimestamp: <TIMESTAMP>"


Signature:
0x4571136f7d2a0ab82fb6da4d6206ff3a8027e3328593b805b0c640e2f35bdbab58121bb2ab0589221c02f665058f396a4bddc5d551629bd3d2c557bf419afc651c
```